progenitor-client/src/progenitor_client.rs

```diff
@@ -416,3 +416,17 @@ impl<E> RequestBuilderExt<E> for RequestBuilder {
             })?))
     }
 }
+
+#[doc(hidden)]
```

Comment:
imo this should be documented

Reply:
Added docstring. I think hiding shrinks the public interface, while client code can reside in a different crate than the generated client lib.